connectors-home/modules/ROOT/pages/windows-gateway-services/windows-gateway-services-guide-mule4.adoc

```diff
@@ -111,46 +115,46 @@ When you configure the connector running in the Mule Runtime, set the authorizat
 
-=== About Impersonating a Windows User Through Custom HTTP Headers
+=== Impersonating a Windows User Through Custom HTTP Headers
 
 Users executing the call on behalf of a Gateway-served connector authenticate through two custom HTTP headers, mule-impersonate-username and mule-impersonate-password.
 
 These two headers represent the Windows credentials of an existing user in the Active Directory forest where the Windows Gateway service is running, or a local account on the machine hosting the service. When these HTTP headers are included in an HTTP Request, the Windows Gateway service authenticates and impersonates this user before executing the operation required by the connector. This provides the ability to configure the correct access control list permissions using Windows credentials.
 
-== About Interaction of MSMQ Connector and Gateway
+== MSMQ Connector and Gateway Interaction
 
 The following diagram shows the interaction of the MSMQ connector with the Gateway, along with the main components used:
 
-image::msmq-windows-gateway.png[Windows Gateway]
+image::msmq-windows-gateway.png["Windows Gateway Services ASP .NET Web API and Mule ESB with Mule runtime and Jersey HTTP client running on JVM on the OS of your choice"]
 
-=== About Configuration Settings
+=== MSMQ Configuration Settings
 
 In the following table you can find the configuration settings that are only related to the MSMQ connector:
 
 [%header,cols="30a,70a"]
 |===
 |Property |Usage
-|invalid-queue-name |The queue name where unreadable messages are moved to.
-|transaction-timeout |The timeout for processing messages since they got retrieved by the connector. When the cleanup task finds a message which timeout expired it moves the message to the main queue to be available back again (for details check two phase commit section at the connector's guide).
-|invalid-message-timeout |The timeout for invalid messages for when a payload of a message is parsed with an incorrect formatter.
-|cleanup-delay |The delay of the cleanup task to start looking for expired messages after they got retrieved for processing (for details check two phase commit section at the connector's guide).
-|cleanup-username |(Optional) The user to impersonate when running the cleanup task. If you choose to leave this setting empty then the user account running the service will be used.
+|invalid-queue-name |The queue name where unreadable messages are moved.
+|transaction-timeout |The timeout for processing messages after they are retrieved by the connector. When the cleanup task finds a message with and expired timeout, it moves the message to the main queue to be so that it's available again. For details, check the two-phase commit section of the connector's user guide.
```

Comment:
```suggestion
|transaction-timeout |The timeout for processing messages after they are retrieved by the connector. When the cleanup task finds a message with an expired timeout, it moves the message to the main queue to be so that it's available again. For details, check the two-phase commit section of the connector's user guide.
```